src/test/java/tech/jhipster/lite/project/infrastructure/secondary/SpiedFileSystemProjectFilesConfiguration.java

```diff
@@ -0,0 +1,42 @@
+package tech.jhipster.lite.project.infrastructure.secondary;
+
+import java.nio.charset.StandardCharsets;
+import org.mockito.Mockito;
+import org.springframework.boot.test.context.TestConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Primary;
+import tech.jhipster.lite.module.infrastructure.secondary.FileSystemProjectFiles;
+
+@TestConfiguration
+public class SpiedFileSystemProjectFilesConfiguration {
+
+  @Bean
+  @Primary
+  public FileSystemProjectFiles fileSystemProjectFiles() {
```

Comment:
Usually beans and spring configuration (class above) don't need to be public, since they're not accessed by other classes.
```suggestion
  FileSystemProjectFiles fileSystemProjectFiles() {
```

Reply:
I will do this 👍